src/app/SubmissionList.jsx

```diff
@@ -149,6 +192,30 @@ const styles = {
       marginBottom: 10,
       boxShadow: '0px 0px 5px 0px rgba(0,0,0,0.2)'
     },
+    pagination: {
```

Comment:
We can fix the height for 10 submissions and leave the pagination bar fixed (it's just a proposal 💅🏻)